### PR TITLE
Bug 1170456 - [Calendar] fix remain jsdoc error

### PR DESCRIPTION
**Pull request by Kiblyn11**

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b379bd7b6ff67d39c322f213047c88a5124b75b1

**Comment by gasolin** [comment]
Please don't use merge, use rebase instead so there's no extra merge log in gaia tree. 
And please squash your PR to single commit.

File: apps/calendar/jsdoc.json [hunk 1]
```diff
@@ -1,15 +1,11 @@
 {
   "name": "calendar",
   "tags": {
-    "dictionaries": ["jsdoc"]
+    "dictionaries": ["jsdoc", "closure"]
```
Comment (gasolin) [inline apps/calendar/jsdoc.json]: why add `closure`?

**Comment by RomainLoisel** [comment]
OK it's my first contribution, I get it for the next.

I add closure to add additional tag support, but it works without closure.

**Pull request closed without merging**